packages/app/src/app/overmind/modals.ts

```diff
@@ -41,6 +41,7 @@ export const alertModal: {
   state: {
     title: string;
     message?: string;
+    customComponent?: 'MemberPaymentConfirmation';
```

Comment:
This should just a string if someone else wants to use it for another modal, no?

Reply:
kinda annoying but you add your component here so that it's checkable with typescript.

this needs to match with "names" that the alert supports

Reply:
You can do:

```
type ModalKeys = keyof typeof modals
```

And you import the modals object from this file: https://github.com/codesandbox/codesandbox-client/blob/fix-closing.issue/packages/app/src/app/pages/common/Modals/index.tsx#L45

I asked Christian about this

Reply:
Strictly typed it now 👍